x/staking/types/keys.go

```diff
@@ -56,8 +56,6 @@ var (
 	ParamsKey = []byte{0x51} // prefix for parameters for module x/staking
 
 	DelegationByValIndexKey = []byte{0x71} // key for delegations by a validator
-
-	NextMigrateDelegationsByValidatorIndexKey = []byte{0x81} // key used to migrate to the new validator index
```

Comment:
Just leaving a comment since I don't remember it from top of my head, is it ok to use same keys in the future per module?

Reply:
Yeah it is, this key has been fully removed from the store now